### PR TITLE
chore(driver-test-support): Bump chai and chai-as-promised

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

**Review by KazuCocoa — COMMENTED** [review]
lgtm so far, hm, ci is not good tho

**Comment by eglitise** [comment]
There's some file changes not related to `chai` - maybe they can be extracted in separate PR(s)?
* `base-driver/lib/express/server.js`
* `execute-driver-plugin/lib/execute-child.js`
* `types/lib/server.ts`

I think this PR is already large enough without these changes :)

**Comment by mykola-mokhnach** [comment]
> There's some file changes not related to `chai` - maybe they can be extracted in separate PR(s)?
> 
> * `base-driver/lib/express/server.js`
> * `execute-driver-plugin/lib/execute-child.js`
> * `types/lib/server.ts`
> 
> I think this PR is already large enough without these changes :)

I would love to do it, but seems like there is no way to make test pass without the above changes. The new chai is ESM and modules need to be updated in order to be compatible with it

**Comment by eglitise** [comment]
If those changes are still compatible with the current chai, maybe they can be extracted and merged before this PR?

**Comment by mykola-mokhnach** [comment]
> If those changes are still compatible with the current chai, maybe they can be extracted and merged before this PR?

Created two separate PRs

File: packages/types/lib/server.ts [hunk 1]
```diff
@@ -47,10 +46,6 @@ export interface AppiumServerExtension {
   webSocketsMapping: Record<string, WSServer>;
 }
 
-export interface AppiumServerSocket extends Socket {
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply packages/types/lib/server.ts:50]: marked it as deprecated in https://github.com/appium/appium/pull/20348

**Review by KazuCocoa — APPROVED** [review]
lg

**Pull request merged**